packages/tools/kadena-cli/package.json

```diff
@@ -47,7 +47,7 @@
     "@kadena/pactjs-generator": "workspace:*",
     "@scure/bip39": "^1.2.1",
     "cardano-crypto.js": "https://github.com/kadena-io/cardano-crypto.js.git#jam@chainweaver-keygen",
-    "chalk": "^4.1.2",
+    "chalk": "^5.2.0",
```

Comment:
lets make sure these changes to the packages don't break anything ( yes we should be up to date, but donno if there are breaking changes )